Content.Server/GameObjects/Components/Conveyor/ConveyorComponent.cs

```diff
@@ -52,13 +53,37 @@ private ConveyorState State
             set
             {
                 _state = value;
+                UpdateAppearance();
+            }
+        }
+
+        public override void Initialize()
+        {
+            base.Initialize();
+
+            if (Owner.TryGetComponent(out PowerReceiverComponent? receiver))
+            {
+                receiver.OnPowerStateChanged += OnPowerChanged;
+            }
```

Comment:
Also unsubscribe (probably in OnRemove and then put this in OnAdd?) at least until we do something about events. Rest LGTM.